prereise/gather/flexibilitydata/test/test_geo_data.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+
+from geo.batch_process import *
```

Comment:
This import should be fixed so that it points to `prereise.gather.flexibilitydata.geo.batch_process` and has explicit imports rather than `import *`.

Reply:
> prereise.gather.flexibilitydata.geo.batch_process

ah I see. I did not install the prereise package so it did not work that way.

Reply:
I _think_ that as long as you're running the tests from the root of the repository, you don't need to have installed the latest version of the code before you run `pytest`, since running at the repo root it should always use the local files first.

In general though, we need the imports to be from an installed package, not a local folder/file.